.codeclimate.yml

```diff
@@ -59,16 +59,6 @@ plugins:
       - XXX
   pep8:
     enabled: true
-  reek:
-    enabled: true
-    exclude_patterns:
-    - 'spec/**/*'
-    - 'db/**/*'
-  rubocop:
```

Comment:
In reference to my other comment - to be clear the idea is we're just getting rid of it in codeclimate, right? Keep it around in the app though?

Reply:
whoooops no I just meant to remove reek and keep rubocop 👮

Reply:
Oh hehe

Reply:
Thanks for catching!